pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -250,6 +255,17 @@ message AuthData {
     optional bytes auth_data = 2;
 }
 
+enum KeySharedMode {
+    autoSplit = 0;
+    exclusiveHashRange = 1;
+}
+
+message KeySharedMeta {
+    required KeySharedMode keySharedMode = 1;
+    required int32 totalHashRange = 2;
```

Comment:
I don't think we should allow users to configure thee totalHashRange. The hash range should be fixed (e.g. 0.0 ~ 1.0, Long.MIN_VALUE ~ Long.MAX_VALUE, or Integer.MIN_VALUE to Integer.MAX_VALUE, or 0 ~ 66536). Just pick up a range, and let consumer specify the hash range. 

Allowing user configuring the `totalHashRange` introduces inconsistent across consumers of a subscription. people can make mistakes.

Reply:
Ok, i will fix by using a fixed range(0-65536), current total range for key shared subscription is also (0-65536)